pymeasure/instruments/keithley/keithley2260B.py

```diff
@@ -0,0 +1,71 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    # truncated_range,
+    # truncated_discrete_set,
+    strict_discrete_set,
+)
+# from pymeasure.adapters import VISAAdapter
+from .buffer import KeithleyBuffer
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Keithley2260B(Instrument):
+    """ Represents the Keithley 2260B Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the socket is hardcoded to 2268,
```

Comment:
I can't find this hard-coding in the source code. Do you maybe mean that the instrument only accepts connections at that port?

Reply:
Yes, if connected via LAN and TCPIP, the instrument accepts connections only at this port (as far as I know), although a webserver can be reached at the usual http port 80. 
This is what I meant with hard-coded: it is hard-coded in the instrument. 
For me, it is usually a bit of a pain to find the correct port, at which an instrument will accept a connection, as (at least with Keithley) the manufacturers often neglect to mention it in the instrument documentation (reference manuals). I know of port scanning, but haven't had time to look up how it works, and many of my colleagues in experimental research fields are either not aware of it altogether, or don't immediately know how to do it either. 

Since this instrument can be connected via other methods too, I did not want to actually hard-code the adapter in pymeasure, but wanted to leave the hint for the socket in the documentation. I will change it to your wording.